src/Fixer/Comment/HeaderCommentFixer.php

```diff
@@ -222,7 +222,29 @@ private function findHeaderCommentCurrentIndex(Tokens $tokens, $headerNewIndex)
     {
         $index = $tokens->getNextNonWhitespace($headerNewIndex);
 
-        return null === $index || !$tokens[$index]->isComment() ? null : $index;
+        if (null === $index || !$tokens[$index]->isComment()) {
+            return null;
+        }
+
+        $next = $index + 1;
```

Comment:
Isn't the convention here in the repo to name this `$nextIndex`?

Reply:
they both exist in many places. I don't know if there is a preferred one.

Reply:
Fair enough